spec/01-unit/01-db/01-schema/11-declarative_config/03-flatten_spec.lua

```diff
@@ -139,6 +140,7 @@ describe("declarative config: flatten", function()
               tls_verify_depth = null,
               tls_verify = null,
               ca_certificates = null,
+              enabled = true,
```

Comment:
Once no longer `required` these `enabled = true` can be removed.

Reply:
I think most of these have to remain as it's checking the result of the declarative config, and the default=true will attach enabled=true.